examples/src/main/java/com/basistech/rosette/examples/SimilarTermsExample.java

```diff
@@ -25,6 +25,9 @@
 
 import java.io.IOException;
 
+/**
```

Comment:
I believe I hit this issue before with topics (and forgot about until now), but the F&F doc generation depends on the existence of this block